lib/sgtn-client/api/translation.rb

```diff
@@ -89,13 +91,18 @@ def self.getTranslation(component, key, locale)
       end
 
       def self.get_cs(component, locale)
-        locale = SgtnClient::LocaleUtil.process_locale(locale)
-        flocale = SgtnClient::LocaleUtil.fallback(locale)
-        cache_key = SgtnClient::CacheUtil.get_cachekey(component, flocale)
+        # source locale always return source bundle
+        if locale == LocaleUtil.get_source_locale
+          sources = SgtnClient::Source.getSources(component, SgtnClient::Config.configurations.default)
+          messages = sources&.first&.last
+          return {'locale' => locale, 'component' => component, 'messages' => messages} if messages
+        end
+
```

Comment:
It's duplicated with similar codes in getStrings
```suggestion

```

Reply:
Will this introduct a problem? If not, I prefer to perceed with current to make changes as small as possible.

Reply:
suggested change is smaller actually, i approve it to speed up the progress.